Kickstarter-iOS/DataSources/LiveStreamContainerMoreMenuDataSource.swift

```diff
@@ -0,0 +1,37 @@
+import Library
+import LiveStream
+import Prelude
+import UIKit
+
+fileprivate enum Section: Int {
```

Comment:
i mean scoped inside the class:

```
class DataSource {
  enum Section {
```

reason being is that we typically scope `Section` to be `internal` so that it can be tested, and so then we need to namespace `Section` so that there aren't conflicts.